myClass.ts

```diff
@@ -0,0 +1,48 @@
+class Myclass {
+	firstNumber: number = 0;
+	lastNumber: number = 0;
+	backend: string = "http://localhost:2050";
+	resizeTimeout: number = 0;
+
+	/** fetches the number of records from backend */
+	fetchRecordCount(): Promise<number> {
```

Comment:
You saaaaay `number`... yet you return `res.json()`... something is not adding up.